src/styles/stars.scss

```
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+  width: 96px;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
```

u can use only 1 property instead - [background](https://developer.mozilla.org/en-US/docs/Web/CSS/background)
```suggestion
    background: ............
```